### DISCUSSION TITLE
celery.exceptions.ImproperlyConfigured: Unknown result backend: 'mysql'.  Did you spell that correctly? ('is a Python module, not a backend class.')

### DISCUSSION THREAD

**Discussion by IKkSssDendi** [apache/airflow#26469] [Q&A]
Python version: 3.8.10
Airflow version: 2.3.4
Here is my part of airflow.cfg
sql_alchemy_conn = mysql://user:pwd@host:3306/db
executor = CeleryExecutor
broker_url = redis://host:6379/0
result_backend = db+mysql://user:pwd@host:3306/db

whatever i use `db+mysql` or 'redis',it always raise celery.exceptions.ImproperlyConfigured: Unknown result backend: 'mysql'.  Did you spell that correctly? ('is a Python module, not a backend class.')

**Comment by potiuk** [comment]
You need to add db+ in front - see https://stackoverflow.com/questions/60740554/airflow-scheduler-crashes-when-setting-postgres-celery-result-backend

**Reply by IKkSssDendi** [reply]
i did，but always

**Reply by IKkSssDendi** [reply]
`db+mysql://airflow:hm_airflow@xxxxxx8:3306/airflow`  like this

**Comment by IKkSssDendi** [comment]
I have read this article [https://stackoverflow.com/questions/60740554/airflow-scheduler-crashes-when-setting-postgres-celery-result-backend](url); `result_backend=db+mysql://airflow:hm_airflow@host:3306/airflow`  However,always report an error on the webserver;

celery.exceptions.ImproperlyConfigured: Unknown result backend: 'mysql'. Did you spell that correctly? ('is a Python module, not a backend class.')

**Reply by potiuk** [reply]
I think your configuration is wrongly set simply. I think you have some configuraiton applied in some way that you are not aware of - you need to find it and fix it. Especially the fact that you get the same`mysql` issue when you modify it sounds like you are just not looking at the right configuration.

I suggest you to experiment - make some spelling mistakes and see if you have erros about the same connection and if you do - find where the configuration is taken from. It can have multiple sources https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html amd you need to find the right one.

**Reply by IKkSssDendi** [reply]
Fine,I'm the one who's being stupid.The problem is not with airflow;This error is caused by a configuration error in my supervisor;
